### ISSUE TITLE
[MED] option to exclude tools that only check contrast

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-evaluation-tools-list#110]
Summary:
* Provide option to exclude tools that only check contrast or simulate color changes.
* Probably also an option to see only those tools.

Rationale:

There are so any tools that only do contrast, that it degrades the user experience. A quick skim of the current list finds ~20+ tools that are only contrast checkers or simulators. And, we continue to get more -- another 5+ that we haven't added.

_(We expect the new list be shorter, since several tools are not longer active. If most of the contrast tools re-submit theirs, we could end up with contrast tools being ~20% of the whole list.)_

We're forcing users who want a more robust tool to read through all those contrast checkers. (And, similarly, someone wanting only a contrast checker to wade through other tools.)

---

https://github.com/w3c/wai-evaluation-tools-list/issues/101 questions the list of what things check, and we might even decide to get rid of that. If we do, we still might want to have a contrast option.

---

side note: We don't want to use "color contrast" alone. From [Easy Checks, Contrast ratio ("color contrast")](https://www.w3.org/WAI/test-evaluate/preliminary/#contrast)
> (This accessibility requirement is sometimes called sufficient "color contrast"; however, that is incorrect — technically it's "luminance contrast". On this page we use "contrast ratio" as short for "luminance contrast ratio" because it's less jargony.)

**Comment by iadawn** [comment]
Does filtering on Accessibility Checks not achieve this?

**Comment by iadawn** [comment]
Given the already high number of filters and the chance that we will be removing the 'Accessibility Checks' I am not inclined to put this in without a strong body of supporting evidence.

**Comment by josehilera** [comment]
It is necessary to use a color contrast analiser tool to evaluate the SC 1.4.11.
Maybe a solution can be include in the list those tools that allow you to pick up two colors direcly from the screen and check if the contrast is correct, but do not include those that require the user to provide color codes.

**Comment by shawna-slh** [comment]
> It is necessary to use a color contrast analiser tool to evaluate the SC 1.4.11.

Right. The point is if there are lots and lots of tools that only check contrast, it adds unnecessary complexity for people to wade through if they want a tool that checks everything.

> Maybe a solution can be include in the list those tools that allow you to pick up two colors direcly from the screen and check if the contrast is correct, but do not include those that require the user to provide color codes.

Thanks for the idea.

I don't see any reason to exclude tools that require the user to provide colors codes.
